layer/profiles.cpp

```diff
@@ -4680,40 +4715,40 @@ VkResult JsonLoader::ReadProfile(const Json::Value root, const std::vector<std::
                 AddPromotedExtensions(properties_api_version);
             }
         } else if (layer_settings->simulate_capabilities & SIMULATE_API_VERSION_BIT) {
-            AddPromotedExtensions(this->profile_api_version);
+            AddPromotedExtensions(this->profile_api_version_);
         }
 
-        if (VK_VERSION_PATCH(this->profile_api_version) > VK_VERSION_PATCH(pdd_.physical_device_properties_.apiVersion)) {
+        if (VK_VERSION_PATCH(this->profile_api_version_) > VK_VERSION_PATCH(pdd_->physical_device_properties_.apiVersion)) {
```

Comment:
`VK_VERSION_PATCH` (actually deprecated, and `VK_API_VERSION_PATCH` should be used instead) is only returning the patch version from the full version. Are you sure that's what you meant to use here and not compare the major+minor version instead?